Firestore/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 - [feature] Added `whereField(arrayContains:)` query filter to find
   documents where an array field contains a specific element.
 - [fixed] Fixed compilation with older Xcode versions (#1517).
+- [fixed] Fixed performance issue when trying to commit a write batch with many
```

Comment:
This is more specific than it needs to be and uses internal terminology. I suggest something like the following phrasing:

> Fixed a performance issue where large write batches with hundreds of changes would take a long time to read and write and consume excessive memory. Large write batches should now see no penalty.

Reply:
Also, please include the issue number/numbers if there are any.

Reply:
Finally, note that you can include the CHANGELOG update in the change that fixes the issue (#1533, right)?

Reply:
Thanks, I like the rephrasing much better. I don't think there was a Github issue for this change, though.

> you can include the CHANGELOG update in the change that fixes the issue

The changes I was trying to describe are #1505 and #1507, already merged.

I've added an entry accounting for #1533 to this PR to avoid merging, though I can also add the relevant entry to #1533 if you prefer.

Reply:
Putting these here is fine--that was most FYI for the future.